### PR TITLE
feat: loyalty leaderboard landingpage

### PR DESCRIPTION
**Pull request by ahnafalfariza**

### CODE REVIEW

File: components/Loyalty/Leaderboard/LeaderboardButton.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+			{/* CHOOSE LB BUTTON DESKTOP*/}
+			<div className="md:flex items-center justify-center gap-3 w-full -mt-12 hidden">
+				<button
+					className={clsx('active:scale-95', activeTab !== 'platinum' && 'opacity-50')}
+					onClick={() => setActiveTab('platinum')}
+				>
+					<img
+						className="pointer-events-none w-48"
+						src="https://paras-cdn.imgix.net/bafkreihqrqmdoh3vj74733tvumf5ukhoxt4miqydwpkmlliulfxmw7ukwq"
```
Comment (iqbalutomo) [inline components/Loyalty/Leaderboard/LeaderboardButton.js:44]: ini gapake parseImgUrl hu?

File: components/Loyalty/LoyaltyTC.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+const LoyaltyTC = () => {
+	return (
+		<div className="bg-white p-8 pl-12 md:pl-16">
+			<p className="text-3xl font-bold mb-2">Terms & Conditions</p>
+			<ul className="list-decimal text-sm md:text-base">
```
Comment (iqbalutomo) [inline components/Loyalty/LoyaltyTC.js:5]: nitip tambahin `text-justify` supaya rapih hu

File: components/Loyalty/Leaderboard/LeaderboardButton.js (hunk 1 above)
Comment (ahnafalfariza) [reply components/Loyalty/Leaderboard/LeaderboardButton.js:44]: gua bikin hardcoded emang hu ini, soalnya kalo pake parseImgUrl di testnet dia hitnya ke ipfs trus kadang ga ngeload gambarnya

File: components/Loyalty/LoyaltyTC.js (hunk 2 above)
Comment (ahnafalfariza) [reply components/Loyalty/LoyaltyTC.js:5]: ooya okok

File: components/Loyalty/Leaderboard/LeaderboardButton.js (hunk 1 above)
Comment (iqbalutomo) [reply components/Loyalty/Leaderboard/LeaderboardButton.js:44]: oh oke hu

**Review by iqbalutomo — DISMISSED** [review]

File: components/Loyalty/LoyaltyTC.js (hunk 2 above)
Comment (ahnafalfariza) [reply components/Loyalty/LoyaltyTC.js]: [9cc80a1](https://github.com/ParasHQ/paras-landing/pull/982/commits/9cc80a18ddf456a31967d6147cc851de5aaf33fe) done gan

**Review by yuesth — APPROVED** [review]

**Review by iqbalutomo — DISMISSED** [review]

**Review by iqbalutomo — APPROVED** [review]

**Review by yuesth — APPROVED** [review]

**Pull request merged**